In [7]:
import numpy as np
from statsmodels.stats.proportion import proportions_ztest

 **Group A: Only notebook sales**

In [ ]:

sales_a =180   
count_a =1000  

**Group B: Notebook + pen sales**

In [10]:
sales_b =230   
count_b =1000

**Convert to arrays**

In [11]:
sales = np.array([sales_a, sales_b])
counts = np.array([count_a, count_b])

**Z-test**

In [12]:
statistic,p_value = proportions_ztest(sales,counts)
print(f"z-statistic: {statistic:.2f}")
print(f"p-value: {p_value:.4f}")

z-statistic: -2.77
p-value: 0.0056


In [13]:
if p_value < 0.05:
    print("The difference is statistically significant — Variant B performs better.")
else:
    print("No statistically significant difference — Variants perform similarly.")

The difference is statistically significant — Variant B performs better.


**Result : The p-value is 0.0056, which is less than 0.05, so Variant B performs significantly better than Variant A.**